TelligentLocalInstance/TelligentLocalInstance.psd1

```diff
@@ -12,7 +12,7 @@
 ModuleToProcess = ''
 
 # Version number of this module.
-ModuleVersion = '0.0.6'
+ModuleVersion = '0.0.7'
```

Comment:
Nit: as this is significant functionality rather than just bug fixes I'd increment the minor version rather than patch, but since we're still pre 1.0, it doesn't hugely matter.